src/s6/debian-root/usr/local/bin/install.sh

```diff
@@ -10,7 +10,7 @@ WEB_LOCAL_REPO=/var/www/html/admin
 setupVars=/etc/pihole/setupVars.conf
 
 detect_arch() {
-  DETECTED_ARCH=$(dpkg --print-architecture)
+  DETECTED_ARCH=$(arch)
```

Comment:
This won't return the same values:

```console
pi@rdweb:~ $ dpkg --print-architecture
armhf

pi@rdweb:~ $ arch
armv7l
```

Reply:
You're right. I tested on two devices I have here and it even simplifies the logic

```
chrko@ThinkPad-X230:~$ arch
x86_64
chrko@ThinkPad-X230:~$ dpkg --print-architecture
amd64
```

```
nanopi@nanopi:~$ dpkg --print-architecture
arm64
nanopi@nanopi:~$ arch
aarch64
```

Reply:
I think there are more possible architectures here.

Testing on a Raspberry Pi 4B running Bullseye I got this:

```console
pi@raspberrypi:~ $ dpkg --print-architecture
arm64

pi@raspberrypi:~ $ uname -m
aarch64

pi@raspberrypi:~ $ arch
aarch64
```